src/makeCard.js

```diff
@@ -12,32 +12,30 @@ function makeCard(episode) {
     cardLine.className = "line-card";
 
     const charactersTitle = document.createElement('h4');
-    charactersTitle.innerHTML = "Personagens: ";
+    charactersTitle.innerHTML = "Characters: ";
   
     const charactersDiv = document.createElement('div');
     charactersDiv.className = "characters";
   
-    // inserting the episode's enumeration, title and air date
-    nameDateDiv.innerHTML = `<h2>${episode.enumeration} - ${episode.title}</h2>
-                             <p>${episode.airDate}</p>`;
+    const characterListEven = document.createElement('ul');
     
-    // inserting characters in the list
-    let listContent = "";
-    episode.characters.forEach( (character, index, array) => {
+    const characterListOdd = document.createElement('ul');
 
-        listContent += `<li>${character}</li>`;
-        
-        // separates the characters in sets of 5
-        if ((index != 0 && (index+1)%5 == 0) || index == array.length-1) {
-            const characterList = document.createElement('ul');
-            characterList.innerHTML = listContent;
-            charactersDiv.appendChild(characterList);
-            listContent = "";
-        }
 
-    });
+    nameDateDiv.innerHTML = `<h2>${episode.episode} - ${episode.name}</h2>
+                             <p>${episode.air_date}</p>`;
     
-    // inserting all elements required in the Card
+
+    episode.characters.forEach( (character, index) => {
+        
+        const correctList = index % 2 === 0 ? characterListEven : characterListOdd;
```

Comment:
You can define an function called *isOdd / is Even* that makes this check and make this code cleaner.